README.md

```diff
@@ -15,7 +15,7 @@
 ![screenshot](./screenshots/screen2.png)
 Covid Tracker is site used to check covid updates.
```

Comment:
Please ensure that the app has unit tests for several components of the app and some pure functions in Redux code according to the project requirement.